### PR TITLE
Use jquery from CDN rather than shipping it

### PR DESCRIPTION
**Pull request by jrha**
Completely untested!

### CODE REVIEW

**Comment by tofu-rocketry** [comment]
Is this the standard way to include jQuery? What are the benefits and risks of doing it this way?

File: unused_resources/test.html [hunk 1]
```diff
@@ -16,44 +16,43 @@
...
 
+        <!-- Latest compiled and minified CSS -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
+        <!-- Optional theme -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css">
+        <!-- Latest compiled and minified JavaScript -->
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
 
-        <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+        <script src="https://code.jquery.com/jquery-1.12.4.min.js" integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ=" crossorigin="anonymous"></script>
```
Comment (GRyall) [inline unused_resources/test.html]: The commit says it is sorting out white space - this is more than white space.

Comment (tofu-rocketry) [reply unused_resources/test.html]: There are two commits in this PR. 7bbbd02 does state what's happening.

**Comment by jrha** [comment]
See https://jquery.com/download/#using-jquery-with-a-cdn for a discussion of the benefits of using the CDN

File: unused_resources/test.html (hunk 1 above)
Comment (jrha) [reply unused_resources/test.html]: Nope, rebase error, fixed now.

**Review by tofu-rocketry — DISMISSED** [review]

**Comment by jrha** [comment]
Rebased on `dev`

**Comment by gregcorbett** [comment]
ta muchly. 

I've put this branch on a [jenky VM](https://host-172-16-103-87.nubes.stfc.ac.uk/portal/) and will perform the usual sanity checks that nothing has broken before merging :)

**Review by tofu-rocketry — APPROVED** [review]

**Pull request merged**